### PR TITLE
Add double extensions and tests

### PR DESCRIPTION
**Pull request by IMBA0**
I've added a couple of double comparison hacks.

### CODE REVIEW

File: CSharpHacks/CSharpHacks.Tests/DoubleTests.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+namespace CSharpHacks.Tests
+{
+    public class DoubleTests
+    {
+        [Fact]
+        public void Zero_should_be_zero()
+        {
+            double value = 0;
+            value.IsZero().Should().Be(true);
```
Comment (madSimonJ) [inline CSharpHacks/CSharpHacks.Tests/DoubleTests.cs]: this is fine, but there's actually a "BeZero()" function available

**Comment by madSimonJ** [comment]
Thanks!

File: CSharpHacks/CSharpHacks.Tests/DoubleTests.cs (hunk 1 above)
Comment (IMBA0) [reply CSharpHacks/CSharpHacks.Tests/DoubleTests.cs]: How would you use it?

In this case I'm testing the method `IsZero()`, which returns bool, so assertion should be comparing bools surely?

**Comment by IMBA0** [comment]
Thanks for merging 👍

File: CSharpHacks/CSharpHacks.Tests/DoubleTests.cs (hunk 1 above)
Comment (madSimonJ) [reply CSharpHacks/CSharpHacks.Tests/DoubleTests.cs]: now I think about it, I'm less sure.  In any case, Fluent Assertions is only for use in unit tests.  Much as we might sometimes like to, we can't use their features in production code

**Pull request merged**